storage/storage_src.go

```diff
@@ -62,9 +74,12 @@ func newImageSource(sys *types.SystemContext, imageRef storageReference) (*stora
 		imageRef:        imageRef,
 		systemContext:   sys,
 		image:           img,
-		layerPosition:   make(map[digest.Digest]int),
 		SignatureSizes:  []int{},
 		SignaturesSizes: make(map[digest.Digest][]int),
+		getBlobMutexProtected: getBlobMutexProtected{
```

Comment:
Nit: Please keep the order of `Signature*Sizes` vs. `getBlobMutexProtected` the same in the struct definition and this constructor. (I don’t care which one.)

Reply:
This was resolved in https://github.com/containers/image/pull/2287.